src/stdune/path.mli

```diff
@@ -316,6 +316,8 @@ val link : t -> t -> unit
 
 val rm_rf : ?allow_external:bool -> t -> unit
 
+val clear : t -> unit
```

Comment:
This function needs  a documentation comment

Reply:
I agree that a doc comment would be nice. Here and in the few other places. (not sure if you marked these resolved by accident or you disagree)